src/objects/media/media.stories.mdx

```diff
@@ -46,49 +47,9 @@ const imageDemoTransformSource = (_src, storyContext) => {
     reverse: false,
   }}
   argTypes={{
+    ...mediaArgTypes,
     imgSrc: { type: { name: 'string', required: true } },
-    tag_name: {
-      type: { name: 'string' },
-      description: 'The root tag for the component',
-      table: {
-        defaultValue: {
-          summary: 'div',
-        },
-      },
-    },
-    inner_tag_name: {
-      type: { name: 'string' },
-      description:
-        'The tag for both the media content and media object elements',
-      table: {
-        defaultValue: {
-          summary: 'div',
-        },
-      },
-    },
-    content_tag_name: {
-      type: { name: 'string' },
-      description: 'The tag for the media content element',
-      table: {
-        defaultValue: {
-          summary: '`inner_tag_name`',
-        },
-      },
-    },
-    object_tag_name: {
-      type: { name: 'string' },
-      description: 'The tag for the media object element',
-      table: {
-        defaultValue: {
-          summary: '`inner_tag_name`',
-        },
-      },
-    },
     text: { type: { name: 'string' } },
```

Comment:
I didn't move this `argType` because it is related to the demo, not the template itself.